config/parameters/sensors/base.txt

```diff
@@ -1,10 +1,23 @@
-# Basic configuration information shared across all sensor model configurations.
+# Basic configuration information shared across all sensor model initial
+# conditions.
 #
 
 # Leader spacecraft base sensor configuration
 
-sensors.leader.gyroscope.bias  0.02 0.01 -0.03
+sensors.leader.gps.r.sigma  0.0 0.0 0.0
+
+sensors.leader.gyroscope.w.bias        0.02    0.01    -0.03
+sensors.leader.gyroscope.w.bias.sigma  1.00e-6 1.00e-6  1.00e-6
+sensors.leader.gyroscope.w.sigma       2.75e-4 2.75e-4  2.75e-4
+
+sensors.leader.magnetometer.b.sigma  5.00e-7 5.00e-7 5.00e-7
```

Comment:
What are the sources for these numbers? Should we link them in this config file so that the source is known?

Reply:
Yep, very good point. Currently, my source was Stewart's attitude estimator MATLAB script but I will see if he linked anything there. If not, I may open up a separate ticket about adding links/documentation for these numbers because I have to focus on other school things for the next few days and then will be travelling and may not have time to fix this PR up before then.

Reply:
Updated a comment at the top of the file. These values were derived experimentally and the documentation in stored in the listed path on Google drive!